coral-component-masonry/examples/index.html

```diff
@@ -283,25 +300,32 @@ <h2 class="coral-Heading--M">Usage notes</h2>
             }
 
             function createImageItem() {
+              var id = getUID();
```

Comment:
Please use commons.getUID to get unique id. https://github.com/adobe/coral-spectrum/blob/master/coral-utils/src/scripts/Commons.js#L555